modules/node/cms_node_api.e

```diff
@@ -249,6 +249,26 @@ feature -- Access: Node
 			end
 		end
 
+	user_is_node_owner (u: READABLE_STRING_32; nid: INTEGER_64): BOOLEAN
```

Comment:
the terminology used by this ROC CMS is "author"
thus, rename user_is_node_owner by

user_is_node_author
and probably, we can just name it "is_node_author (u: ...)"

As well, we should review all query names, and see when we use nid, or node object, similar uid and user object.